tests/kafkatest/services/verifiable_consumer.py

```diff
@@ -48,19 +50,27 @@ def handle_shutdown_complete(self):
         self.assignment = []
         self.position = {}
 
-    def handle_offsets_committed(self, event):
+    def handle_startup_complete(self):
+        self.state = ConsumerState.Started
+
+    def handle_offsets_committed(self, event, node, logger):
         if event["success"]:
             for offset_commit in event["offsets"]:
+                if "error" in offset_commit and offset_commit["error"] != "":
```

Comment:
python style nit: You can also shorten tests like this to `if offset_commit.get('error')`. `get` defaults to `None` whereas the index operator will throw an exception if the key doesn't exist.